include/sym1.h

```diff
@@ -0,0 +1,133 @@
+#define DISP_P         0x73   // 'P'
+#define DISP_q         0x67   // 'q'
+#define DISP_r         0x50   // 'r'
+#define DISP_S         0x6D   // 'S'
+#define DISP_t         0x46   // 't'
+#define DISP_U         0x3E   // 'U'
+#define DISP_u         0x1C   // 'u'
+#define DISP_V_1       0x64   // 'V'
+#define DISP_V_2       0x52   // 2nd half
+#define DISP_W_1       0x3C   // 'W'
+#define DISP_W_2       0x1E   // 2nd half
+#define DISP_Y         0x6E   // 'Y'
+#define DISP_Z         0x5B   // 'Z'
+#define DISP_SPACE     0x00   // ' '
+#define DISP_PERIOD    0x80   // '.'
+#define DISP_HYPHEN    0x40   // '-'
+#define DISP_APOSTR    0x20   // '''
+#define DISP_EQUAL     0x41   // '='
+#define DISP_3_BAR     0x49   // '='
+#define DISP_BOTTOM    0x08   // '_'
+#define DISP_TOP       0x01   // Top segment
+#define DISP_LEFT      0x30   // '|' Left side, both segments
+#define DISP_RIGHT     0x06   // '|' Right side, both segments
+#define DISP_DEGREE    0x63   // 'o' An 'o' character in the upper segments
+#define DISP_HAT       0x23   // 'n' An 'n' character in the upper segments
+#define DISP_FORK      0x62   // 'u' A  'u' character in the upper segments
+#define DISP_SLASH     0x51   // '/'
+#define DISP_BACKSLASH 0x34   // '\'
+#define DISP_TOP_RIGHT 0x02   // Top right segment
+#define DISP_TOP_LEFT  0x20   // Top left segment
+#define DISP_LOW_RIGHT 0x04   // Lower right segment
+#define DISP_LOW_LEFT  0x10   // Lower left segment
+
+
+/*****************************************************************************/
+/*                                 Hardware                                  */
+/*****************************************************************************/
+
+
+
+#include <_6522.h>
+#define VIA1    (*(struct __6522*)0xA000)    // U25
+#define VIA2    (*(struct __6522*)0xA800)    // U28
+#define VIA3    (*(struct __6522*)0xAC00)    // U29
+
+
+struct _display {
+    unsigned char d0;   // left-most seven-segment display
+    unsigned char d1;   // second seven-segment display
+    unsigned char d2;   // third seven-segment display
+    unsigned char d3;   // fouth seven-segment display
+    unsigned char d4;   // fifth seven-segment display
+    unsigned char d5;   // sixth seven-segment display
+    unsigned char d6;   // buffer byte to the right
+};
+#define DISPLAY (*(struct _display*)0xA640)
+
+
+
+/*****************************************************************************/
+/*                                   Code                                    */
+/*****************************************************************************/
+
+
+
+void beep (void);                                                  // Beep sound
+void fdisp (void);                                                 // Flash display
+int __fastcall__ loadt (unsigned char);                            // Read from tape (id)
+int __fastcall__ dumpt (unsigned char, const void*, const void*);  // Write to tape (id, start_addr, end_addr)
```

Comment:
Please adhere to the cc65 code style. Here:

```
void beep (void);
/* Beep sound */

void fdisp (void);
/* Flash display */

int __fastcall__ loadt (unsigned char id);
/* Read from tape */

int __fastcall__ dumpt (unsigned char id, const void* start_addr, const void* end_addr); 
/* Write to tape */
```

Reply:
Updated.  Please check and confirm.